test/functional/autoupdate/aup-basic.bats

```diff
@@ -53,9 +53,7 @@ test_teardown() {
 	assert_is_output "$expected_output"
 
 	# check for files(should not exist)
-	run sudo sh -c "test -c $PATH_PREFIX/etc/systemd/system/swupd-update.service"
-	assert_status_is_not "$SWUPD_OK"
-	run sudo sh -c "test -c $PATH_PREFIX/etc/systemd/system/swupd-update.timer"
-	assert_status_is_not "$SWUPD_OK"
+	assert_file_not_exists "$PATH_PREFIX/etc/systemd/system/swupd-update.service"
+	assert_file_not_exists "$PATH_PREFIX/etc/systemd/system/swupd-update.timer"
```

Comment:
don't you also need to run "autoupdate" again to check for the correct status after enabling it?

Reply:
I didn't add it initially as it would have created confusion. But I see why not. Please let me know if that explanation looks good for it

Reply:
oh so there is not way to use autoupdate to get the status when using a --path? I'm confused now 😛